charts/osm/crds/meshconfig.yaml

```diff
@@ -81,6 +81,10 @@ spec:
                       description: Global list of IP address ranges to exclude from outbound traffic interception by the sidecar proxy.
                       type: string
                       pattern: ((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2})(,((?:\d{1,3}.){3}\d{1,3})\/(\d{1,2}))*$
+                    outboundPortExclusionList:
+                      description: Global list of ports to exclude from outbound traffic interception by the sidecar proxy.
+                      type: integer
+                      pattern: ^[1-9]\d*$
```

Comment:
The type here is integer. Does it only support one port rather than a list?

Reply:
This should be a list of integers. However, outboundIPRangeExclusionList should also be a list of strings. This needs to be fixed with the latest CRD API schema.

Reply:
@snehachhabria since I have a pending PR migrating to MeshConfig, do you want me to fix the port exclusion list altogether?

Reply:
@allenlsy yes please, if you could fix it as a part of that changes it would be great